charmcraft/commands/build.py

```diff
@@ -353,16 +354,22 @@ def pack_charm_in_instance(
             target_arch=get_host_architecture(),
         )
         base_configuration = providers.get_base_configuration(
-            alias=build_base,
+            alias=build_base_alias,
             instance_name=instance_name,
         )
 
+        if build_on.name != "ubuntu":
+            allow_unstable = True
+            emit.message(f"Will use {build_on.name} on unstable daily image for build.")
```

Comment:
Can you more explicitly warn the user that CentOS is a "use at your own risk" base?

I'm thinking something like `Warning: Base {build_on.name} {build_on.channel} daily image may be unstable.`

Reply:
OK